### ISSUE TITLE
Freeze in com.googlecode.gtalksms/.panels.wizard.Wizard (ANR keyDispatchingTimedOut)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#207]
```
3 reports (see attachments)

```

Original issue reported on code.google.com by `Florent....@gmail.com` on 1 Aug 2011 at 9:57

Attachments:
- [GTalkSMS Freeze 1.txt](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-207/comment-0/GTalkSMS Freeze 1.txt)
- [GTalkSMS Freeze 2.txt](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-207/comment-0/GTalkSMS Freeze 2.txt)
- [GTalkSMS Freeze 3.txt](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-207/comment-0/GTalkSMS Freeze 3.txt)

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 2 Aug 2011 at 8:40
- Added labels: **Wizard**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 31 Aug 2011 at 10:59
- Changed title: **Freeze in com.googlecode.gtalksms/.panels.wizard.Wizard (ANR keyDispatchingTimedOut)**
- Added labels: **FromMarket**

**Comment by GoogleCodeExporter** [comment]
```
Working on a new user interface that will fix these issues
```

Original comment by `Florent....@gmail.com` on 20 May 2012 at 12:09
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
No more wizard with the new UI
```

Original comment by `Florent....@gmail.com` on 10 Jun 2012 at 4:01
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with GTalkSMS v4.0 and released on market
```

Original comment by `Florent....@gmail.com` on 28 Jul 2012 at 4:20
- Changed state: **Deployed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with GTalkSMS v4.0 and released on market
```

Original comment by `Florent....@gmail.com` on 28 Jul 2012 at 4:20
